Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandardV3/README.md

```diff
@@ -0,0 +1,4728 @@
+                "id": "dummy_id",
+                "is_system": false,
+                "name": "Abhi-CB-policy",
+                "num_devices": 1,
+                "position": 2,
+                "priority_level": "LOW",
+                "total_num_devices": 1
+            },
+            {
+                "description": "Don't worry about this. It won't be here long.",
+                "id": "dummy_id",
+                "is_system": false,
+                "name": "Splunk Policy To_be_deleted_1652997758",
+                "num_devices": 0,
+                "position": 39,
+                "priority_level": "MEDIUM"
+            },
+            {
+                "description": "Splunk Policy invalid value",
+                "id": "dummy_id",
+                "is_system": false,
+                "name": "Splunk Policy invalid value1652962163",
+                "num_devices": 0,
+                "position": 42,
+                "priority_level": "HIGH"
+            },
+            {
+                "description": "aaaaaaaa",
+                "id": "dummy_id",
+                "is_system": false,
+                "name": "test42",
+                "num_devices": 0,
+                "position": 107,
+                "priority_level": "MEDIUM"
+            }
+        ]
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Policies summaries
+>|Id|Name|Priority Level|Is System|
+>|---|---|---|---|
+>| 80947 | AWN BAS | MEDIUM | false |
+>| 87816 | Abhi-CB-policy | LOW | false |
+>| 103786 | CBCLOUD Policy | MEDIUM | false |
+>| 33819 | Cigent Policy | HIGH | false |
+>| 68727 | Confluera Policy 1 | LOW | false |
+>| 69390 | Cyborg_Monitor | LOW | false |
+>| 102049 | DarkNegev | MEDIUM | false |
+>| 12147 | DefenseStorm Policy | MEDIUM | false |
+>| 109968 | Devo-MP-Skoville | MEDIUM | false |
+>| 105029 | EasyNAC domain | MEDIUM | false |
+>| 35704 | Fortress Policy | MISSION_CRITICAL | false |
+>| 130711 | Hunters High Test | HIGH | false |
+>| 41528 | Hunters Policy | MEDIUM | false |
+>| 128135 | Hunters Policy test | MISSION_CRITICAL | false |
+>| 99682 | Iron-policy | MEDIUM | false |
+>| 63139 | LRDemo-JH | MEDIUM | false |
+>| 138424 | Lions (Victor+Long) | MISSION_CRITICAL | false |
+>| 9246 | LogRhythm Policy | MEDIUM | false |
+>| 104163 | Lumu | MEDIUM | false |
+>| 65120 | Lumu Policy | LOW | false |
+>| 96535 | Moja Skopska polisa | LOW | false |
+
+
+### cbd-create-policy
+
+***
+Creates a new policy on the CB Defense backend.
+
+#### Base Command
+
+`cbd-create-policy`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| description | A description of the policy. Can be multiple lines. | Optional | 
+| name | A unique one-line name for the policy. | Optional | 
+| priorityLevel | The priority score associated with sensors assigned to this policy. Possible values: "MISSION_CRITICAL", "HIGH", "MEDIUM", and "LOW". Possible values are: MISSION_CRITICAL, HIGH, MEDIUM, LOW. | Optional | 
```

Comment:
Delete the extra Possible values: "MISSION_CRITICAL", "HIGH", "MEDIUM", and "LOW".